### PR TITLE
feat: add - & m7b5 qualities

### PR DESCRIPTION
**Pull request by dok**
### What's new

This PR adds the chord qualities "-" & "m7b5" to the list.

### Why the change?

"m7b5" is a pretty common chord in Jazz

### CODE REVIEW

File: pychord/constants/qualities.py [hunk 1]
```diff
@@ -25,6 +26,7 @@
     ('7sus4', (0, 5, 7, 10)),
     ('m6', (0, 3, 7, 9)),
     ('m7', (0, 3, 7, 10)),
+    ('m7b5', (0, 3, 5, 10)),
```
Comment (yuma-m) [inline pychord/constants/qualities.py:29]: I think `m7b5` or `m7(b5)` is as same as `m7-5`, isn't it?

```suggestion
    ('m7-5', (0, 3, 6, 10)),
```

File: pychord/constants/qualities.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
     # 3 notes
     ('', (0, 4, 7)),
     ('maj', (0, 4, 7)),
+    ('-', (0, 3, 7)),
```
Comment (yuma-m) [inline pychord/constants/qualities.py:12]: Could you swap this line with the next? The more common notations are listed above in this OrderedDict.

File: pychord/constants/qualities.py (hunk 1 above)
Comment (dok) [reply pychord/constants/qualities.py:29]: Ah. I wasn't sure whether those symbols are the same. Looks like there are some usages for `b5` & `-5` in some places like here: https://github.com/dok/pychord/blob/663b84e981a7113f07865aefc8a3c98f1d760dee/pychord/constants/qualities.py#L22-L23

File: pychord/constants/qualities.py (hunk 2 above)
Comment (dok) [reply pychord/constants/qualities.py:12]: Yes!

**Pull request merged**